05-expressions.tex

```diff
@@ -220,6 +220,12 @@ \section{for}
 
 Variable \expr{v} is then available within loop body \expr{e2} and holds the value of the individual elements of collection \expr{e1}.
 
+The closest Haxe for-loops come to C-style for-loops is probably using the range operator \expr{...}. It is a binary operator taking two \expr{Int} arguments: \expr{min...max} returns an \expr{IntIterator} instance that iterates from \expr{min} (inclusive) to \expr{max} (exclusive). Note that \expr{max} may not be smaller than \expr{min}.
```

Comment:
I mainly mentioned C-for-loops because it's the first sentence of this section ("Haxe does not support traditional for-loops known from C.").

- Ok.
- Are [Numeric operators](http://haxe.org/manual/types-numeric-operators.html) an exception then?
- It seems ok to say that an operator returns something, though? (again, that's the term used on the numeric operators page)